lib/widgets/drop_down.dart

```diff
@@ -1,42 +1,52 @@
 import 'package:dropdown_button2/dropdown_button2.dart';
 import 'package:flutter/material.dart';
 
-class DropDown extends StatefulWidget {
-  List<String> items;
+class DropDown extends StatelessWidget {
+  final String? dropdownValue;
+  final List<String> items;
+  final Function didSelectItemHandler;
 
-  DropDown(this.items, {super.key});
+  const DropDown(
+      {required this.items,
+      required this.didSelectItemHandler,
+      this.dropdownValue,
+      super.key});
 
   @override
-  State<DropDown> createState() => _DropDownState();
-}
-
-class _DropDownState extends State<DropDown> {
-  String dropdownValue = "swift";
-  @override
-  Widget build(BuildContext context) => DropdownButton2(
-        isExpanded: true,
-        hint: Text(
-          'Select Item',
-          style: TextStyle(
-            fontSize: 14,
-            color: Theme.of(context).hintColor,
+  Widget build(BuildContext context) => Container(
+        margin: const EdgeInsets.only(top: 10, bottom: 10),
+        padding: const EdgeInsets.all(10),
+        decoration: BoxDecoration(
+          color: Theme.of(context).colorScheme.secondary,
+          borderRadius: BorderRadius.circular(20),
+          border: Border.all(
+            color: Colors.white,
+            width: 1,
+          ),
+        ),
+        child: DropdownButton2(
```

Comment:
What was the rationale behind importing dropdown_button2 package? Is there no other way we could do a dropdown (besides building it ourselves)? Like from Material package? Or a different design pattern?

Reply:
Tried with the local one but its position of showing dropdown varies based on the previously selected index. So used this to show always show dropdown from the bottom of the button